hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestRatisPipelineCreateAndDestroy.java

```diff
@@ -103,7 +108,7 @@ public void testPipelineCreationOnNodeRestart() throws Exception {
     } catch (IOException ioe) {
       // As now all datanodes are shutdown, they move to stale state, there
       // will be no sufficient datanodes to create the pipeline.
-      Assert.assertTrue(ioe instanceof InsufficientDatanodesException);
+      Assert.assertTrue(ioe instanceof SCMException);
```

Comment:
So all the policies including ContainerPlacementPolicy are throwing out SCMException with specified reason in messages. The PipelinePlacementPolicy follows them and so as upstream SCMPipelineManager. So here I align the old InsufficientDatanodesException with SCMException. I believe with specified error message, it's ok to use a general exception since the test is specifically testing pipeline creation.